lib/pact/consumer/configuration/configuration_extensions.rb

```diff
@@ -70,6 +70,22 @@ def pactfile_write_mode= pactfile_write_mode
           @pactfile_write_mode = pactfile_write_mode
         end
 
+        def pactfile_write_order
+          @pactfile_write_order ||= :chronological #or :alphabetical
+        end
+
+        def pactfile_write_order= pactfile_write_order
+          @pactfile_write_order = pactfile_write_order.to_sym
+        end
+
+        def interactions_replay_order
```

Comment:
Hey Alex. Thanks for adding the config options. It looks good, but the `interactions_replay_order` belongs in lib/pact/provider/configuration/configuration_extensions.rb